tests/session_tests.py

```
@@ -5,34 +5,50 @@
-                "-u",
-                os.path.join(
-                    os.path.dirname(os.path.abspath(__file__)),
-                    "utils",
-                    "session_helper.py",
-                ),
-            ]
-            + list(args)
-        )
-        .decode()
-        .replace("\r", "")
+def _make_session_shell(start_time, disable_warning):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_make_session_shell`

(ANN202)